dlink-admin/src/main/java/com/dlink/service/impl/DataBaseServiceImpl.java

```diff
@@ -75,7 +75,11 @@ public boolean saveOrUpdateDataBase(DataBase dataBase) {
             return false;
         }
         if (Asserts.isNull(dataBase.getId())) {
-            checkHeartBeat(dataBase);
+            try {
+                checkHeartBeat(dataBase);
+            } catch (Exception e) {
+                throw new RuntimeException(e);
```

Comment:
Does throw exception still prevent code from saving function? I thought the PR want to keep the save function running all the time.

Reply:
Yes, when your configuration information is abnormal, it should not affect the saving of the data source, but my pr has a problem. I need to modify it again